runtime/common/src/paras_registrar.rs

```diff
@@ -69,6 +68,9 @@ pub trait Config: paras::Config + dmp::Config + ump::Config + hrmp::Config {
 	/// Runtime hook for when a parachain and parathread swap.
 	type OnSwap: crate::traits::OnSwap;
 
+	/// Runtime hook for when a parachain should be cleaned up.
```

Comment:
What does "cleaned up" mean here? Documentation could do to be expanded on this somewhere.